src/test/java/org/cactoos/collection/SlicedTest.java

```diff
@@ -41,19 +41,22 @@
 
     @Test
     @SuppressWarnings("PMD.AvoidDuplicateLiterals")
-    public void sliceCollection() {
+    public void sliceCollection() throws IOException {
+        final Collection<String> expected = new CollectionOf<>(
+            "three", "four"
+        );
         new Assertion<>(
-            "Can't get slice portion of collection",
+            new FormattedText(
+                "Must get sliced collection of [%s]",
+                expected
+            ).asString(),
```

Comment:
@fanifieiev I'm not sure why you bothered complexifying things, simply rewording the reason was enough. Can you revert to a simpler tests here and for the other please?